### ISSUE TITLE
Error during activating

### ISSUE THREAD

**Issue by jdneo** [testforstephen/vscode-pde#40]
Contains: Bundle startup failed reference:file:/c:/Users/xxx/.vscode/extensions/yaozheng.vscode-pde-0.6.0/server/org.sonatype.tycho.m2e_0.8.1.201704211436.jar

org.osgi.framework.BundleException: Could not resolve module: org.sonatype.tycho.m2e [126]

  Unresolved requirement: Require-Bundle: org.eclipse.m2e.core; bundle-version="[1.4.0,2.0.0)"

**Comment by rgrunber** [comment]
FWIW, I think I've seen something similar :

```
!ENTRY org.sonatype.tycho.m2e 4 0 2022-06-10 11:42:39.728
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.sonatype.tycho.m2e [123]
  Unresolved requirement: Require-Bundle: org.eclipse.m2e.core; bundle-version="[1.4.0,2.0.0)"

    at org.eclipse.osgi.container.Module.start(Module.java:463)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel$2.run(ModuleContainer.java:1847)
    at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor$1$1.execute(EquinoxContainerAdaptor.java:136)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1840)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1781)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1745)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1667)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
    at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)

!ENTRY org.eclipse.jdt.ls.core 1 0 2022-06-10 11:42:40.436
!MESSAGE class org.eclipse.jdt.ls.core.internal.JavaLanguageServerPlugin is started

!ENTRY org.eclipse.jdt.ls.core 1 0 2022-06-10 11:42:40.645
!MESSAGE Main thread is waiting

!ENTRY org.eclipse.jdt.ls.core 1 0 2022-06-10 11:42:40.683
!MESSAGE >> initialize

!ENTRY org.eclipse.jdt.ls.core 1 0 2022-06-10 11:42:40.686
!MESSAGE Initializing Java Language Server 1.13.0.202206090105
...
...
```

In another project, I see :
```
message: '[Error - 2:10:52 p.m.] Jun. 10, 2022, 2:10:52 p.m. Failed to load extension bundles \n' +
    'Load bundle list\n' +
    'org.eclipse.core.runtime.CoreException: Load bundle list\n' +
    '\tat org.eclipse.jdt.ls.core.internal.handlers.BundleUtils.loadBundles(BundleUtils.java:169)\n' +
...
...
'\tat java.base/java.lang.Thread.run(Unknown Source)\n' +
    'Contains: Bundle startup failed reference:file:/home/rgrunber/.vscode/extensions/yaozheng.vscode-pde-0.6.0/server/org.sonatype.tycho.m2e_0.8.1.201704211436.jar\n'
```

**Comment by testforstephen** [comment]
yes, I'm also seeing this exception. It seems org.sonatype.tycho.m2e bundle is not maintained by a few years. So far, I didn't find any feature breaking with this exception. I will double check again, if this bundle is not needed, I may consider removing it directly from vscode-pde extension.

**Comment by testforstephen** [comment]
Addressed in 0.7.0
